fs/xfstests.py.data/btrfs/4k.yaml

```diff
@@ -12,6 +12,6 @@ loop_type: !mux
 fs_type: !mux
     fs_btrfs_4k:
         fs: 'btrfs'
-        args: '-R xunit -g quick'
-        mkfs_opt: '-f -s 4096 -n 4096'
+        args: '-R xunit -L 10 -g quick'
+        mkfs_opt: '-f -s 4096 -n 4096 -O quota'
```

Comment:
no please. Keep quota mkfs option separate. 
4k.yaml/64.yaml  are meant to have default configurations tested. For testing quota - please create separate file.